src/app/src/actions/auth.js

```diff
@@ -136,11 +149,37 @@ export function sessionLogin() {
 }
 
 export function submitForgotPassword() {
-    return (dispatch) => {
+    return (dispatch, getState) => {
         dispatch(startSubmitForgotPassword());
 
-        return Promise
-            .resolve(true)
+        const {
+            auth: {
+                forgotPassword: {
+                    form: {
+                        email,
+                    },
+                },
+            },
+        } = getState();
+
+        if (!email) {
+            return dispatch(logErrorAndDispatchFailure(
+                null,
+                'Email address was not provided',
+                failSubmitForgotPassword,
+            ));
+        }
+
+        if (!isEmail(email)) {
+            return dispatch(logErrorAndDispatchFailure(
+                null,
+                'Email address is invalid',
+                failSubmitForgotPassword,
```

Comment:
That makes sense. I renamed this entire set of Redux actions to be `...forgotPasswordEmail` to indicate that it describes the whole process of preparing, sending, and completing the request rather than just simply making the request. 53e8cbac31975632fafbb88e4d85f41af9c5f0ab